content/using_bookwyrm/main-menu.md

```diff
@@ -0,0 +1,123 @@
+---
+Title: Main Menu & Timelines
+Date: 2025-09-06
+Order: 1
+---
+
+## Your Books view
+
+In the top-left of the page (or top of the page on mobile) you will find a small selection of your books. 
+It will include up to 5 books, from your Currently Reading, Read and To Read reading status shelves.
+ 
+To see all of them, you may need to *horizontally* scroll, or pull on the horizontal scroll bar.
+
+If you have 3 or 4 currently reading, the To Read section will not be displayed.
+Similarly, if all 5 are currently reading, the Read section won't be available either.
+However, if you have more than 5 currently reading, it will still only show 5 of them.
+
+By clicking on one of these books, it becomes the focus of the interact and review section. 
+
+### Interact & Review section
+
+Below the display of your books is a section that displays basic information about the selected book, and lets you interact with it.
+It includes:
+
+- The book's title, linking to it's main page.
+- The name of the author, linking to the author's page.
+- The [update reading status](/shelves.html#updating-reading-status) button for that book.
+- Tabs to select a type of status you might want to [post about the book](/posting-statuses.html).
+The options are Review, Comment or Quote.
```

Comment:
This should be on the end of the line above, otherwise your unordered list won't display properly.